Sources/MapboxCoreNavigation/PassiveLocationDataSource.swift

```diff
@@ -64,66 +53,8 @@ open class PassiveLocationDataSource: NSObject {
     /**
      Starts the generation of location updates with an optional completion handler that gets called when the location data source is ready to receive snapped location updates.
      */
-    public func startUpdatingLocation(completionHandler: ((Error?) -> Void)? = nil) {
+    public func startUpdatingLocation() {
```

Comment:
This public API change is worth a mention in the changelog.

Reply:
Makes sense. Added.

Reply:
The documentation above it was not updated. It still says optional completion handler.

Reply:
Also, previously an error was passed in the completion handler. Now, the function doesn't throw an error. Is it good to assume no errors will be raised here?

Reply:
Sorry for the inconsistent code comment, we'll update it.
> Is it good to assume no errors will be raised here?

Reply:
Thanks a lot